### ISSUE TITLE
Zoom in/out loops when in car navigation mode

### ISSUE THREAD

**Issue by Altonss** [organicmaps/organicmaps#3142]
When in car navigation mode, the map zooms in and out automatically, which is a cool feature but most of the time it just zooms in and out on a fairly straight road with no apparent reason, resulting in a zoom in/out loop repeating every few seconds which is annoying. (And the whole map is updating/flickering all details because of names: too much house number/street names/POI disappearing while 0.5s and then reappearing.

**Comment by biodranik** [comment]
Isn't it a duplicate? Did you search for other similar issues?
Zooming should depend on your current speed.

**Comment by Altonss** [comment]
I searched for duplicates, but didn't find an issue that matched. If someone finds already opened issues about this topic, I would be happy to close this one :)


> Zooming should depend on your current speed.

Yes that seems logic, but somehow small variations of the speed should not trigger weird zoom in/zoom out loops...

**Comment by Echolon** [comment]
> I searched for duplicates, but didn't find an issue that matched. If someone finds already opened issues about this topic, I would be happy to close this one :)
> 
> > Zooming should depend on your current speed.
> 
> Yes that seems logic, but somehow small variations of the speed should not trigger weird zoom in/zoom out loops...

Yes, I want to confirm this. It can be actually dangerous as it leads to distraction while driving. I assume this results from weird mapping / street layouts in openstreetmap.
